authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/exception/IdAuthExceptionHandler.java

```diff
@@ -1,426 +1,434 @@
+			kycAuthResponseDTO.setErrors(errors);
+			kycAuthResponseDTO.setResponseTime(responseTime);
+			//kycAuthResponseDTO.setResponse(kycResponseDTO);
+			return kycAuthResponseDTO;
+		case "otp":
+			OtpResponseDTO otpResponseDTO = new OtpResponseDTO();
+			otpResponseDTO.setErrors(errors);
+			otpResponseDTO.setResponseTime(responseTime);
+			return otpResponseDTO;
+		case "internal":
+			if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.OTP)) {
+				OtpResponseDTO internalotpresponsedto = new OtpResponseDTO();
+				internalotpresponsedto.setErrors(errors);
+				internalotpresponsedto.setResponseTime(responseTime);
+				return internalotpresponsedto;
+			} else if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.AUTH_TRANSACTIONS)) {
+				AutnTxnResponseDto autnTxnResponseDto = new AutnTxnResponseDto();
+				autnTxnResponseDto.setErrors(errors);
+				autnTxnResponseDto.setResponseTime(responseTime);
+				return autnTxnResponseDto;
+			} else if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.AUTH_TYPE)) {
+				AuthtypeResponseDto authtypeResponseDto = new AuthtypeResponseDto();
+				authtypeResponseDto.setErrors(errors);
+				authtypeResponseDto.setResponseTime(responseTime);
+				return authtypeResponseDto;
+			}else if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.DECRYPT)) {
+				AuthtypeResponseDto authtypeResponseDto = new AuthtypeResponseDto();
+				authtypeResponseDto.setErrors(errors);
+				authtypeResponseDto.setResponseTime(responseTime);
+				return authtypeResponseDto;
+			}else if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.ENCRYPT)) {
+				AuthtypeResponseDto authtypeResponseDto = new AuthtypeResponseDto();
+				authtypeResponseDto.setErrors(errors);
+				authtypeResponseDto.setResponseTime(responseTime);
+				return authtypeResponseDto;
+			}else if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.PUBLICKEY)) {
+				AuthtypeResponseDto authtypeResponseDto = new AuthtypeResponseDto();
+				authtypeResponseDto.setErrors(errors);
+				authtypeResponseDto.setResponseTime(responseTime);
+				return authtypeResponseDto;
+			}else if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.VERIFY)) {
+				AuthtypeResponseDto authtypeResponseDto = new AuthtypeResponseDto();
+				authtypeResponseDto.setErrors(errors);
+				authtypeResponseDto.setResponseTime(responseTime);
+				return authtypeResponseDto;
+			}else if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.VALIDATESIGN)) {
+				AuthtypeResponseDto authtypeResponseDto = new AuthtypeResponseDto();
+				authtypeResponseDto.setErrors(errors);
+				authtypeResponseDto.setResponseTime(responseTime);
+				return authtypeResponseDto;
+			}else if (Objects.nonNull(type) && type.equalsIgnoreCase(IdAuthCommonConstants.HOTLIST)) {
+				AuthtypeResponseDto authtypeResponseDto = new AuthtypeResponseDto();
```

Comment:
@mamta-344 , please return ResponseWrapper object instead of AuthtypeResponseDto. because, AuthtypeResponseDto  is for AuthTypeStatus service..

Reply:
I changed the code